src/endpoints/remove.rs

```diff
@@ -17,6 +17,12 @@ pub async fn remove(data: web::Data<AppState>, id: web::Path<String>) -> HttpRes
             .append_header(("Location", format!("{}/", ARGS.public_path)))
             .finish();
     }
+    
+    if ARGS.non_removable {
+        return HttpResponse::Found()
+        .append_header(("Location", format!("{}/", ARGS.public_path)))
+        .finish();
+    }
```

Comment:
Since this if-clause has the same body as the preceding one, why not merge them like this?

```rust
if ARGS.readonly || ARGS.non_removable {
    return HttpResponse::Found()
        .append_header(("Location", format!("{}/", ARGS.public_path)))
        .finish();
}
```